results.go

```diff
@@ -122,6 +124,34 @@ func (v *vocdoniHandler) finalizeElectionResults(election *api.Election, electio
 			return
 		}
 		log.Infow("final results image built ondemand", "electionID", election.ElectionID.String())
+		hubResults := &communityhub.HubResults{
+			Question:   electiondb.Question,
+			Options:    choices,
+			Date:       election.EndDate.String(),
+			CensusRoot: election.Census.CensusRoot,
+			CensusURI:  election.Census.CensusURL,
+		}
+		hubResults.Tally = make([][]*big.Int, 0)
+		for _, option := range election.Results[0] {
+			opt := new(big.Int).Set(option.MathBigInt())
+			hubResults.Tally[0] = append(hubResults.Tally[0], opt)
+		}
+		// hubResults.Turnout = big.NewInt(int64(electiondb.Turnout))
+		// hubResults.TotalVotingPower = big.NewInt(int64(election.Census.MaxCensusSize))
```

Comment:
I don't think we have the total voting power, so we should store it somewhere when creating the census.